Examples/Swift/CustomViewController.swift

```diff
@@ -138,10 +138,10 @@ class CustomViewController: UIViewController, MGLMapViewDelegate, AVSpeechSynthe
             let line = MGLLineStyleLayer(identifier: layerIdentifier, source: lineSource)
 
             // Style the line
-            line.lineColor = MGLStyleValue(rawValue: UIColor(red:0.00, green:0.45, blue:0.74, alpha:0.9))
-            line.lineWidth = MGLStyleValue(rawValue: 5)
-            line.lineCap = MGLStyleValue(rawValue: NSValue(mglLineCap: .round))
-            line.lineJoin = MGLStyleValue(rawValue: NSValue(mglLineJoin: .round))
+            line.lineColor = NSExpression(forConstantValue: UIColor(red:0.00, green:0.45, blue:0.74, alpha:0.9))
+            line.lineWidth = NSExpression(forConstantValue: 5)
+            line.lineCap = NSExpression(forConstantValue: NSValue(mglLineCap: .round))
```

Comment:
nit: `NSExpression(forConstantValue: "round")` seems to be the recommended way according to the documentation and we use throughout the migration except for here.

Reply:
Fixed in 0294f1209b08f3c7329bd0eecb5752aea359b1a7.